v1.1/training/why-cockroachdb.md

```diff
@@ -5,8 +5,27 @@ sidebar_data: sidebar-data-training.json
 redirect_from: /training/why-cockroachdb.html
 ---
 
+Kick off your training by watching Alex Robinson, a CockroachDB Engineer, explain the history of databases and why CockroachDB was built.
```

Comment:
Nit: I'd put this sentence inside the Video div.

Reply:
Same as above.